executor/aggregate.go

```diff
@@ -510,6 +508,9 @@ func (w *HashAggFinalWorker) consumeIntermData(sctx sessionctx.Context) (err err
 		sc               = sctx.GetSessionVars().StmtCtx
 	)
 	for {
+		if w.hashAgg.stats != nil {
+			atomic.AddInt64(&w.hashAgg.stats.FinalTaskNum, 1)
```

Comment:
What does the `FinalTaskNum` mean here?
It seems that `FinalTaskNum` will be `concurrency` at most?

Reply:
Because the finalworker obtaining the result from the partitalworker is a pipeline operation, this parameter is mainly used to record how many times the finalworker task for obtaining the result has been performed